packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -103,7 +104,16 @@ export default decorate([
                   pool.default_SR
                 ),
               })
-            )
+              const oldTemplate = find(
+                templates,
+                template =>
+                  pool.$pool === template.$pool &&
+                  template.other['xo:resource:namespace'] === namespace
```

Comment:
Please add an explanation about what the namespace means and what the id means.

Reply:
We talked about the placement of the documentation not for the id and namespace meaning.

Reply:
`ID` and `NAMESPACE` is a part of `hub documentation`
If you we should comment properties, we should do this in all cases. For example, adding comments to describe `POOL` or `VM` properties.